src/bot/functions/rotate-barrel-to.js

```diff
@@ -27,22 +29,16 @@ module.exports = function(degreeAngle, callback) {
 
 function rotateBarrelTo(angle, elapsedMs) {
   let r = this.barrel.rotationVelocity * elapsedMs;
-  if ((this.barrel.rotation < angle && this.barrel.rotation + r >= angle) || (this.barrel.rotation > angle && this.barrel.rotation + r <= angle)) {
+  if ((this.barrel.rotation <= angle && this.barrel.rotation + r >= angle) || (this.barrel.rotation >= angle && this.barrel.rotation + r <= angle)) {
```

Comment:
The function wasn't finishing when `angle == this.barrel.rotation` 

Instead, it would increment and end up making a full rotation of the barrel